scst-store/logs.hbs.md

```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output 
```

Comment:
I think the wording is a bit unclear. If I'm understanding it correctly, how about something like "Every request received will produce a `Processing request` log, which includes the name of the operation called and the fields that are being requested"